ambry-router/src/test/java/com/github/ambry/router/GetManagerTest.java

```diff
@@ -534,6 +545,18 @@ private void compareBlobInfo(BlobInfo blobInfo, long segmentSize) {
     Assert.assertArrayEquals("User metadata should match", putUserMetadata, blobInfo.getUserMetadata());
   }
 
+  /**
+   * Compare the size of chunk BlobId list with the expected chunk size + metadata chunk.
+   * @param chunkBlobIds List of chunk blob Ids for the composite blob.
+   */
+  private void compareChunkBlobIds(long blobSize, List<StoreKey> chunkBlobIds) {
+    if (blobSize <= CHUNK_SIZE) {
+      Assert.assertNull("chunkBlobIds should be null for simple blob", chunkBlobIds);
+    } else {
+      Assert.assertEquals("the size of chunk BlobId list should match", blobSize / CHUNK_SIZE + 1, chunkBlobIds.size());
```

Comment:
Is this always true ? There could be a 4MB composite blob comprising four 1-MB chunks.

Reply:
For all these tests used this getBlobAndCompareContent method, both stitched blob and composite blob has larger size than the one chunk. So I use this way to make the test simple. Let me know if you have any other preference.